File: Source/ArrayRelatedOperations/invert_array.c
#include <stdio.h>

void invertArray(int a[], int n) {
    int start = 0;
    int end = n - 1;
    while (start < end) {
        int temp = a[start];
        a[start] = a[end];
        a[end] = temp;
        start++;
        end--;
    }
}

int main() {
    int a[] = {1, 2, 3, 4, 5};
    int n = sizeof(a) / sizeof(a[0]);
    
    printf("Original array: ");
    for (int i = 0; i < n; i++) {
        printf("%d ", a[i]);
    }
    
    invertArray(a, n);
    
    printf("\nInverted array: ");
    for (int i = 0; i < n; i++) {
        printf("%d ", a[i]);
    }
    
    return 0;
}

File: Source/ArrayRelatedOperations/invert_array.js
function reverseArray(a) {
    let start = 0;
    let end = a.length - 1;

    while (start < end) {
        let temp = a[start];
        a[start] = a[end];
        a[end] = temp;

        start++;
        end--;
    }
}
